airbyte-ci/connectors/base_images/tests/conftest.py

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+import platform
+import sys
+
+import dagger
+import pytest
+
+
+@pytest.fixture(scope="module")
+def anyio_backend():
+    return "asyncio"
+
+
+@pytest.fixture(scope="module")
+async def dagger_client():
+    async with dagger.Connection(dagger.Config(log_output=sys.stderr)) as client:
+        yield client
+
+
+@pytest.fixture(scope="session")
+def current_platform():
```

Comment:
Could use a few tests and a github action to pair it with!

Let me know if thats another stack

Reply:
Yep I'll a tests in a new stack